Custom Channels (#710)
SUSE/doc-susemanager: modules/administration/pages/channel-management.adoc

Channels are a method of grouping software packages.

In {productname}, channels are grouped into base and child channels, with base channels grouped by operating system type, version, and architecture, and child channels being compatible with their related base channel.
When a client has been assigned to a base channel, it is only possible for that system to install the related child channels.
Organizing channels in this way ensures that only compatible packages are installed on each system.

Software channels use repositories to provide packages.

> The channels mirror the repositories in {productname}, and the packages and other data is stored in the {productname} database.

Reviewer: ```suggestion
The channels mirror the repositories in {productname}, and the package names and other data are stored in the {productname} database.
```
For clarity.

You can have any number of repositories associated with a channel.

> The software from those repositories is then able to be installed on clients by subscribing the client to the appropriate channel.

Reviewer: ```suggestion
The software from those repositories can then be installed on clients by subscribing the client to the appropriate channel.
```

Clients can only be assigned to one base channel.

> The client is then able to install or update packages from the repositories associated with that base channel and any of its child channels.

Reviewer: ```suggestion
The client can then install or update packages from the repositories associated with that base channel and any of its child channels.
```
